packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -56,6 +57,20 @@ export class SelectPageObject {
         return (this.selectElement.selectedOptions[0] as ListOption) ?? null;
     }
 
+    public getActiveOption(): ListOption | null {
+        return (
+            (this.selectElement.options.find(
+                o => (o as ListOption).activeOption === true
```

Comment:
nit: You shouldn't need the `=== true` check here since `activeOption` is already a `boolean`.